<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PuttingblAide</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>串口助手</string>
         </attribute>
         <widget class="QTextBrowser" name="text_rx">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>230</y>
            <width>151</width>
            <height>72</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_clear_rx">
             <property name="text">
              <string>清除接收</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_open">
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btn_clear_tx">
             <property name="text">
              <string>清除发送</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_send">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>20</y>
            <width>143</width>
            <height>169</height>
           </rect>
          </property>
          <property name="title">
           <string>串口设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>串口选择</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_Port"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_Baud">
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>256000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>460800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>921600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2000000</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_Stop">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_DataBit">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>奇偶校验</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboBox_Varity">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>奇</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>偶</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTextEdit" name="text_tx">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>261</width>
            <height>61</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>图像</string>
         </attribute>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>241</width>
            <height>141</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>文件处理</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>75</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>添加头部
信息</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>aes加密</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>161</width>
            <height>101</height>
           </rect>
          </property>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>40</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>21</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>key</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditKey">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>0</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>70</y>
             <width>161</width>
             <height>41</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>iv</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditIv">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>10</y>
             <width>53</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>AES 配置</string>
           </property>
          </widget>
         </widget>
         <widget class="QTextBrowser" name="textFileInfo">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>130</y>
            <width>171</width>
            <height>71</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>文件另存</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_5">
         <attribute name="title">
          <string>IAP升级</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>开始升级</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_Port_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>69</width>
            <height>21</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>串口</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蓝牙</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WiFi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USB-HID</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>48</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>连接方式</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>48</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>文件名称</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>48</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>文件版本</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>70</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>10</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>31</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>协议</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>复位</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>40</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>帧大小</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>40</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>70</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>校验方式</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>70</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
